### PR TITLE
feat(inlineLoading): add component

### PR DESCRIPTION
**Pull request by loganmccaul**
Adds the Inline Loading Component.

Definitely a WIP, so feel free to be critical about any of the content, or let me know what's missing.

### CODE REVIEW

File: src/content/components/inline-loading/style.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+## Color
+
+| Attribute              | SCSS          | HEX      |
+|--------------------|---------------|----------|
+| Spinner      | $ui-05        | #5a6872 |
```
Comment (marijohannessen) [inline src/content/components/inline-loading/style.md]: We've changed this on the website now to reflect the actual CSS class names. We haven't deployed it yet, but you can take a look at one of the other style pages on the master branch to see what I mean. Do you mind updating?

Comment (marijohannessen) [reply src/content/components/inline-loading/style.md]: You can see an example here https://github.com/IBM/design-system-website/blob/master/src/content/components/button/style.md

Comment (loganmccaul) [reply src/content/components/inline-loading/style.md]: Updated the tables, let me know if it's correct or not 😄

**Comment by loganmccaul** [comment]
@marijohannessen I'm not a huge fan on the image quality, and recommendations on image sizes I should export so they come out a bit less pixelated?

**Comment by marijohannessen** [comment]
Hmm @aagonzales , what do you think?

**Comment by aagonzales** [comment]
@loganmccaul you should be exporting images at 2x.

**Comment by loganmccaul** [comment]
@aagonzales Ended up exporting at 4x to get it to not double in size when displayed.

**Comment by alisonjoseph** [comment]
Overall this looks great! Can you add a space between inline and loading on the page. Also, can you link the codepen here: https://codepen.io/team/carbon/full/oMpMYK/ .  You just need to add `oMpMYK` to the `code.md` file.

screen shot 2018-07-30 at 1 43 32 pm

**Comment by asudoh** [comment]
@alisonjoseph Would you want to see if your review comment has been addressed...? Thanks!

**Pull request merged**